packages/story-editor/src/components/floatingMenu/elements/shared/useUpdateSelected.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../../app';
+
+function useUpdateSelected() {
+  const { updateSelectedElements } = useStory(
+    ({ actions: { updateSelectedElements } }) => ({ updateSelectedElements })
+  );
```

Comment:
Oh nice. didn't know `updateSelectedElements` existed as an action. now seeing that it's functionality embedded into the `updateElements` action. Was gonna create this action for perf in some areas, but nice to see it's already there to utilize.